### PR TITLE
wrapper: rename entrypoint to help sourcemaps

### PR DESCRIPTION
**Pull request by 2color**
## What

- Rename entrypoint file to @aragon/wrapper

## Why

- For some strange reason Chrome and Firefox don't seem to be able to parse the source map properly when debugging `aragon/aragon` with `@aragon/wrapper` linked. 
- This extends to our production deployments took, which are deployed with the sourcemaps. Yet for some strange reason, this module isn't picked up properly. See in screenshot below how the module is missing.
Screen Shot 2019-04-30 at 10 25 29 am

### CODE REVIEW

**Comment by 2color** [comment]
@bpierre 
Thanks for pointing this out. We should apply this change to the other packages. 

I'm curious what the root cause of this bug is. 

For example, our production build which includes the sourcemap seems to mix files from the `aragon/aragon` repository and `@aragon/wrapper` package. See below the `cache/index.js` and the `apps` folder, each of which are coming from different repos.

Screen Shot 2019-05-06 at 12 28 32 pm

**Comment by sohkai** [comment]
I think the Chrome debugger is likely confused with the source map paths, since we export them transpiled in the aragon.js repo. I imagine it's having a hard time knowing that it's in `node_modules`, when the source map probably says everything is from `../src/index.js` :shrug:.

File: packages/aragon-wrapper/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@aragon/wrapper",
   "version": "5.0.0-rc.3",
   "description": "Library for Aragon client implementations",
-  "main": "dist/index.js",
+  "main": "dist/aragon-wrapper.js",
```
Comment (sohkai) [inline packages/aragon-wrapper/package.json]: What do you think about keeping an `index.js` file that just re-exports the other modules, to be more similar to other npm packages?

We also do some re-exporting in the current `index.js` that could be removed from `aragon-wrapper.js`.

**Comment by sohkai** [comment]
And on `size-limit`, we should probably specify a glob instead:

```
"path": "dist/*.js"
```

File: packages/aragon-wrapper/package.json (hunk 1 above)
Comment (2color) [reply packages/aragon-wrapper/package.json]: I like the idea. I also had mixed feelings about this given all the other npm packages I've seen.

**Comment by 2color** [comment]
> And on size-limit, we should probably specify a glob instead:
> "path": "dist/*.js"

I'm not sure what you're actually suggesting. size-limit expects an entry point and for every entrypoint we have a size limit so I'm not sure what we'll get with a glob?

**Comment by sohkai** [comment]
Ahh, didn't realize `size-limit` was smart enough to compute all the dependencies from a single entry point :+1:.

**Pull request closed without merging**